README.md

```diff
@@ -1 +1,138 @@
-# leaderboard
\ No newline at end of file
+# To Do List 📝
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="120"  height="auto"/>
+  <br/>
+
+</div>
+
+## A solo programming project that heighlights the use of webpack to bundle JavaScript, ES6 syntax
+
+# 🛠 Built With
+
+### Tech Stack
+
+- HTML5.
+- CSS3.
+- Javascript using ES6.
+- Webpack.
+
+### Client
+
+- HTML.
+- CSS.
+- JS.
+- Webpack
+
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+[To Do List 📝](https://ilyassatlassi.github.io/webpack/)
+
```

Comment:
- Kindly remove the current live demo link, as it's for another project and add the current one or add coming soon.. if you don't have one yet. 👍🏽